content/en/user-guide/aws/eks/index.md

```diff
@@ -551,6 +549,27 @@ spec:
   restartPolicy: Always
 ```
 
+## Supported Versions
+
+LocalStack uses [k3s](https://github.com/k3s-io/k3s) under the hood for creating EKS clusters.
+Below is the list of supported Kubernetes versions and their corresponding k3s versions.
+The default version is `1.22`.
+
+| Kubernetes Version | k3s Version       |
```

Comment:
I presume this list of supported versions comes from the source code? My concern is keeping this list up to date with the supported versions.

We are already 2 releases behind the current release in both the source code _and_ this table. Is there any way we can keep these two up to date?